My first pull request module 1 week 2

mrcbq opened the pull request:
Module 1 week 2 PR

mrcbq committed:
added a linter config in local and for the github actions

mrcbq committed:
Readme File customized and changed

mrcbq committed:
HTML added

mrcbq committed:
Added the class name for the header h1 tag

mrcbq committed:
Added the styles for the class title for the heading tag

mrcbq committed:
.gitignore modified for excluded test.md file

braincee commented on README.md:
```diff
@@ -207,15 +178,15 @@ Example:
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mrcbq)
+<!-- 
 👤 **Author2**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) -->
```
[OPTIONAL] Please since you are the only author. I suggest you remove the second author commented

mrcbq replied:
OK its Done.

braincee commented on README.md:
```diff
@@ -137,7 +108,7 @@ Example command:
 ### Setup
 
-Clone this repository to your desired folder:
+Clone this repository to your desired folder and change the readme file for your fits.
```
[OPTIONAL] Please add a specific setup to the README file

Example
` https://github.com/mrcbq/MS-Module-1.git`

mrcbq replied:
It's Done.

braincee reviewed (changes requested):
Hello @mrcbq  ✋, I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close](https://user-images.githubusercontent.com/78104544/225492039-ed22bba9-481a-489f-ae04-74623970af20.gif)

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Professional README file ✅

REQUIRED CHANGES ♻️
------

- [x] Please make sure all linters passed as it is part of the project requirements.
- [x] Please always add a short summary of changes introduced in the pull request.

OPTIONAL CHANGES
--------

**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

- [x] [OPTIONAL] Please ensure that the commit messages are imperative tenses and 
        capitalized. Example: Please say Add... not Added...
- [x] [OPTIONAL] Please this is an HTML and CSS project. Kindly add a specific installation 
       process `npm install` to the installation section in the README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mrcbq committed:
Readme.md:Delete the second Author info in the readme file

mrcbq committed:
Styles.css:
Correct the indentation of 2 spaces that causs erroor for StyleLint.
indedx.html:
Title changed.

mrcbq commented:
Correct the indentation error that causes errors with the StyleLint.
Delete the lighthouse linter because it seems that the errors were the origin of an internal error that can't connect to the chrome browser.
Using StyleLint and Github Actions as external linter.
Delete the second Author in the readme.md file and add the setup info.

mrcbq committed:
Add a configuration file for styleLint

mrcbq committed:
change in style.css

mrcbq committed:
Correct the styles.css for the stylesLint

mrcbq committed:
Correct the styles.css for the stylesLint

mrcbq committed:
Correct the styles.css for the stylesLint

mrcbq committed:
Add the .github/workflows/linter.yml config file and changed the node version from 18 to 12

mrcbq committed:
Add the .hintrc for webjhint error

mrcbq committed:
Review the error weblint error

mrcbq committed:
Deleted the package-lock.json file

mrcbq committed:
Modiifed the html-checker

mrcbq committed:
Add another html file

mrcbq committed:
Delete the hello.html file test

mrcbq committed:
Change the main in package.json file

mrcbq committed:
Modified the node version for the weblinter from 12.x to 16.x

mrcbq commented:
correct all errors with the linters.
Add all the corrections suggested.
I have several errors with the linters conf, but it seems that all were origin in the node version that use in the linters conf file.

braincee reviewed (approved):
Hi @braincee  , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

HIGHLIGHTS
---

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

![youre-approved-shirley-auto-group](https://user-images.githubusercontent.com/78104544/226852457-95ef4c1d-5d51-4f76-abd8-1fdc06b89e77.gif)

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ ] [OPTIONAL] I kindly suggest making your commit messages imperative tense and capitalized. 

![226624434-f307e11a-3a3a-4b72-bf72-95c7ac702293](https://user-images.githubusercontent.com/78104544/227068459-09baea51-e35a-4d6e-90b8-2f3fd86c5162.png)

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
